frontend/app/components/wp-activity/user/user-activity-directive.ts

```diff
@@ -126,8 +127,14 @@ function userActivity($uiViewScroll,
           wpCacheService.loadWorkPackageLinks(scope.workPackage, 'activities');
           scope.inEdit = false;
         });
+        this.focusEditIcon();
       };
 
+      scope.focusEditIcon = function () {
+        // Find the according edit icon and focus it
+        jQuery('textarea.edit-comment-text').closest(".user-comment").siblings('.comments-number').find('.icon-edit').closest("a").focus();
```

Comment:
That selector seems overly complex and brittle. Can you add an additional class to the comment number link there to help break it down?